src/bls/bls.h

```diff
@@ -127,6 +127,11 @@ class CBLSWrapper
         return impl.Serialize(specificLegacyScheme);
     }
 
+    std::array<uint8_t, SerSize> ToBytes(const bool specificLegacyScheme) const
+    {
+        return impl.SerializeToArray(specificLegacyScheme);
```

Comment:
```suggestion
        if (!fValid) {
            return std::array<uint8_t, SerSize>{};
        }
        return impl.SerializeToArray(specificLegacyScheme);
```

it works for me with this fix 5b4b333958a2816c520e701cf65b427ea05e8a87